src/ContractInteract/EIP20CoGateway.js

```diff
@@ -1,12 +1,40 @@
+/**
+ * @typedef EIP20CoGatewaySetupConfig
+ *
+ * @property {string} valueToken The value token contract address.
+ * @property {string} utilityToken The utility token address that will be used for
+ *                 minting the utility token.
+ * @property {string} anchor Address of anchor to use for getting the state root of
```

Comment:
```suggestion
 * @property {string} stateRootProvider Address of contract that implements the state root provider interface.
```

Also in other parts of the code below, e.g. `deploy`